run.py

```diff
@@ -65,8 +78,16 @@ def main():
 
     inp = "The answer to life the universe and everything is of course"
     print(f"Output for prompt: {inp}", sample_from_model(gen, inp, max_len=100, temperature=0.01))
+  # Add authentication
+  @app.route("/inference")
+  @auth.login_required
```

Comment:
This is simply a stub, and should not be merged.

Reply:
This has not been resolved. It appears the contributor is likely using AI to create this PR, and forgot to implement this method. Not suitable for production, nonetheless.